src/stdafx.h

```diff
@@ -63,6 +63,7 @@
 #include "boost/uuid/uuid_io.hpp"
 #include "boost/uuid/uuid_generators.hpp"
 
+#define PICOJSON_USE_INT64
```

Comment:
This is no longer standard JSON, and I'd prefer to keep it standard. Use double instead.